### PR TITLE
Update local storage documentation

### PR DESCRIPTION
**Pull request by jsafrane**
- link to official origin repository instead of my fork
- add parameter with image name so enterprise customers may use Red Hat shipped and signed images

This partly fixes https://bugzilla.redhat.com/show_bug.cgi?id=1540848

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jsafrane LGTM. Thank you!

**Comment by rootfs** [comment]
lgtm

**Comment by jsafrane** [comment]
@rootfs, @ahardin-rh: I added a new commit that adds documentation about SELinux. PTAL and merge.

This fixes https://bugzilla.redhat.com/show_bug.cgi?id=1542868

File: install_config/configuring_local.adoc [hunk 1]
```diff
@@ -132,35 +139,47 @@ Before starting the provisioner, mount all local devices and create a ConfigMap
 with storage classes and their directories.
 ====
 
+
 Install the local provisioner from the link:https://raw.githubusercontent.com/jsafrane/origin/local-storage/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml[*_local-storage-provisioner-template.yaml_*] file.
 
-. Create a service account that allows running pods as a root user and use HostPath volumes:
+. Create a service account that allows running pods as a root user, use HostPath volumes and use any SELinux context to be able to monitor, manage and clean local volumes:
```
Comment (ahardin-rh) [inline install_config/configuring_local.adoc]: manage, and clean

File: install_config/configuring_local.adoc [hunk 2]
```diff
@@ -132,35 +139,47 @@ Before starting the provisioner, mount all local devices and create a ConfigMap
...
 . Install the template:
 +
 [source, bash]
 ----
-$ oc create -f https://raw.githubusercontent.com/jsafrane/origin/local-storage/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml
 ----
 
-. Instantiate the template by specifying values for `configmap` and `account` parameters:
+. Instantiate the template by specifying values for `configmap`, `account` and `provisioner_image` parameters:
```
Comment (ahardin-rh) [inline install_config/configuring_local.adoc]: 'account`, and

File: install_config/configuring_local.adoc [hunk 3]
```diff
@@ -132,35 +139,47 @@ Before starting the provisioner, mount all local devices and create a ConfigMap
...
 +
 [source, bash]
 ----
 $ oc create serviceaccount local-storage-admin
-$ oc adm policy add-scc-to-user hostmount-anyuid -z local-storage-admin
+$ oc adm policy add-scc-to-user privileged -z local-storage-admin
 ----
-Root privileges are required for the provisioner pod for allowing it to delete
-content on local volumes. HostPath is required to access the
+Root privileges and any SELinux context are required for the provisioner pod for allowing it to delete
```
Comment (ahardin-rh) [inline install_config/configuring_local.adoc]: To allow the provisioner pod to delete content on local volumes created by any pod, root privileges and any SELinux context are required.

**Comment by ahardin-rh** [comment]
@jsafrane Just minor style nits from me. Otherwise, LGTM! We'll need to squash all changes into one commit before we can merge, per our publishing process. Thanks!

**Comment by jsafrane** [comment]
Fixed and squashed. PTAL again.

**Comment by ahardin-rh** [comment]
LGTM!

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**